From 33360fb781f5653f3e7c1db9f25a01a86d51571d Mon Sep 17 00:00:00 2001
From: Piotr Tomaszewski <piotr.tomaszewski@intel.com>
Date: Wed, 13 Nov 2013 16:20:00 +0100
Subject: Add support for composite fstab

Adds support for composing fstab. This makes it
possible to compose fstab in various mixins. It is useful
as Android only reads one fstab file, and it must be called
fstab.{ro.hardware}.

For: ABRO-262
Category: AOSP Improvement
Domain: AOSP-build
Origin: Internal
Upstream-Candiate: Yes

Change-Id: I1240c29ee18047fe754aeecfb055ef94be6f5459
Depends-Change-Id: I6cabc13ad29183425369484e46740c89cc55f980
Depends-Change-Id: I1471ea8ad289efa7e328af38405ebff21305554e
---
 core/Makefile |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/core/Makefile b/core/Makefile
index 7cd78a5..5cd941b 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -357,6 +357,7 @@ DEFAULT_KEY_CERT_PAIR := $(DEFAULT_SYSTEM_DEV_CERTIFICATE)
 $(call assemble-composite-file,init.rc,init.composite.rc)
 $(call assemble-composite-file,ueventd.rc,ueventd.composite.rc)
 $(call assemble-composite-file,init.recovery.rc,init.recovery.composite.rc)
+$(call assemble-composite-file,fstab,fstab.$(TARGET_PRODUCT))
 
 # -----------------------------------------------------------------
 
-- 
1.7.4.1

